DRAFT_climate_indicators.Rmd

```diff
@@ -0,0 +1,1255 @@
+
+```{r setup, include=FALSE}
+library(knitr)
+knitr::opts_chunk$set(echo = TRUE)
+#library(tidyverse) # cant update to v2 on server
+library(dplyr)
+library(ggplot2)
+library(stars)
+library(sf)
+library(tmap)
+library(lubridate)
+library(ggpubr)
+library(microbenchmark)
+library(tictoc)
+library(ncmeta)
+library(parallel)
+library(readr)
+library(exactextractr)
+library(tidyr)
+library(data.table)
+library(terra)
+dir <- substr(getwd(), 1,2)
+
+```
+
+```{r, echo=F}
+Ecosystem <- "All" # e.g. "Skog og fjell"
+Egenskap  <- "Abiotiske egenskaper" # e.g. "Primærproduksjon"
+ECT       <- "Physical state characteristics" # e.g. "Structural state characteristic"
+Contact   <- "Anders L. Kolstad" # e.g. "Anders Kolstad"
+```
+
+```{r, echo=F}
+metaData <- data.frame(Ecosystem,
+                       "Økologisk egenskap" = Egenskap,
+                       "ECT class" = ECT)
+knitr::kable(metaData)
+```
+
+<br /> <br />
+
+<hr />
+
+## Introduction
+
+This chapters describes the development of a workflow for generating or preparing indicators based on interpolated climate data from [SeNorge](https://senorge.no/).
+
+## About the underlying data
+
+The data is in a raster format and extends back to 1957 in the form of multiple interpolated climate variables. The spatial resolution is 1 x 1 km.
+
+### Representativity in time and space
+
+The data includes the last normal period (1961-1990) which defines the reference condition for climate variables. Therefore the temporal resolution is very good. Also considering the daily resolution of the data.
+
+Spatially, a 1x1 km resolution is sufficient for most climate variables, esp. in homogeneous terrain, but this needs to be evaluation for each variable and scenario specifically.
+
+### Original units
+
+Varied. Specified below for each parameter.
+
+### Temporal coverage
+
+1957 - present
+
+### Additional comments about the data set
+
+The data format has recently changed from .BIL to .nc (netcdf) and now a single file contains all the rasters for one year (365 days), and sometimes for multiple variables also.
+
+## Ecosystem characteristic
+
+### Norwegian standard
+
+These variables typically will fall under the *abiotiske egenskaper* class.
+
+### SEEA EA
+
+In SEEA EA, these variables will typically fall under A1 - Physical state characteristics.
+
+## Collinearities with other indicators
+
+Climate variables are most likely to be correlated with each other (e.g. temperature and snow). Also, some climate variables are better classed as pressure indicators, and these might have a causal association with several condition indicators.
+
+## Reference condition and values
+
+### Reference condition
+
+The reference condition for climate variables is defined as the last normal period 1961-1990.
+
+### Reference values, thresholds for defining *good ecological condition*, minimum and/or maximum values
+
+-   Un-scaled indicator value = median value over 5 year periods
```

Comment:
See comment below: 5-year period makes intuitive sense, but may need justification?

Reply:
added justification : (5 years being a pragmatic choice. It is long enough to smooth out a lot of inter-annual variation, and it's long enough to enable estimating errors)